orchestra/frontend/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@b12/metronome": "^1.0.0",
+    "@reduxjs/toolkit": "1.3.0",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^25.1.4",
+    "@types/node": "^13.9.3",
+    "@types/react": "^16.9.25",
+    "@types/react-dom": "^16.9.5",
+    "axios": "0.19.2",
+    "moment-timezone": "0.5.28",
+    "node-sass": "4.13.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-redux": "7.2.0",
+    "react-router-dom": "5.1.2",
+    "react-scripts": "3.4.1",
+    "redux": "4.0.5",
+    "redux-thunk": "2.3.0",
+    "typescript": "^3.8.3",
+    "watch": "^1.0.2"
+  },
+  "scripts": {
+    "start": "react-app-rewired start",
+    "build": "react-app-rewired build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "watch": "watch 'yarn build' ./src --wait=1"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "homepage": "/static/dist2/",
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin-tslint": "2.25.0",
+    "customize-cra": "^0.9.1",
```

Comment:
related to hack, i believe. i'm curious if it will work in production

Reply:
Any way I can go about testing it? maybe i can hijack staging and try out if it works

Reply:
We can punt to after we merge/create an rc version bump.